addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -979,6 +1007,20 @@ const actions = {
         dispatch('closeChatWindow', oldChatWindowLocalId);
         dispatch('focusChatWindow', newChatWindowLocalId);
     },
+    /**
+     * @param {Object} param0
+     * @param {function} param0.dispatch
+     * @param {Object} param0.state
+     * @param {string} composerLocalId
+     */
+    resetComposer({ dispatch, state }, composerLocalId) {
+        Object.assign(state.composers[composerLocalId], {
+            textInputContent : '',
```

Comment:
no space before `:` in object definition

Reply:
Done